### DISCUSSION TITLE
No 2A boot files from 2.6 hotfix 2

### DISCUSSION THREAD

**Discussion by 112021179** [quintenvandamme/nothing_archive#5] [Q&A] [answered]
Haven't been able to find boot files for this particular version anywhere, as well as the full OTA for it to extract the images from payload.bin 
Is there a specific reason for that?

**Comment by quintenvandamme** [comment, marked as answer]
It hasn't been uploaded at https://reindex-ot.github.io/nothing/

**Comment by 112021179** [comment]
Updated to hotfix 2 - no boot files to patch for magisk, can't update to OB 3.0 - have to be on 1906
Bruh
